velox/exec/tests/JoinFuzzer.cpp

```diff
@@ -140,11 +140,15 @@ class JoinFuzzer {
 JoinFuzzer::JoinFuzzer(size_t initialSeed)
     : vectorFuzzer_{getFuzzerOptions(), pool_.get()} {
   filesystems::registerLocalFileSystem();
+
+  // Make sure not to run out of open file descriptors.
+  const std::unordered_map<std::string, std::string> hiveConfig = {
+      {connector::hive::HiveConfig::kNumCacheFileHandles, "1000"}};
   auto hiveConnector =
       connector::getConnectorFactory(
           connector::hive::HiveConnectorFactory::kHiveConnectorName)
-          ->newConnector(kHiveConnectorId, std::make_shared<core::MemConfig>());
-  connector::registerConnector(hiveConnector);
```

Comment:
@mbasmanova We need to add this line back: https://app.circleci.com/pipelines/github/facebookincubator/velox/44041/workflows/b6d7d2f1-a458-47b9-803c-116c5261387a/jobs/307138

Reply:
Oops... not sure how it got deleted. Let me add it back.